### ISSUE TITLE
Space Suite JetPack chestplate doesn't keep enchantments.

### ISSUE THREAD

**Issue by Fireztonez** [MJRLegends/ExtraPlanets#361]
### ---Issue Report---
**(PLEASE DONT DELETE THIS TEMPLATE, YOUR ISSUE WILL BE IGNORED AND CLOSED IF YOU DO.)**

**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):** Yes and I found nothing about that issue...

### Description of Issue
I always enchant my space  suite with Protection and other similar enchant question to be prepared for the Boss Fight, but when try the Space Suite JetPack, and like always apply my enchantments from the anvil on the Space suite, the chestplate loose the enchantments directly the first time I try to fly with it (Even in creative, when the chestplate doesn't use RF).

### Steps to Reproduce Issue
1. Enchant your Jetpack Space Suite Chestpalte
2. Fly
3. Look your chestplate :)

**Crash log/Log (if applicable):** [PASTEBIN. Do NOT copy and paste the log into the issue post itself.]
There have no crash logs, becose there have no crash ;)

**Screenshots (if applicable):** 
![jetpack-space-suite-1](https://user-images.githubusercontent.com/19277301/47033544-21824680-d143-11e8-8407-4f3d64a0297a.png)
![jetpack-space-suite-2](https://user-images.githubusercontent.com/19277301/47033548-23e4a080-d143-11e8-9d67-d96c7019b57c.png)
![jetpack-space-suite-3](https://user-images.githubusercontent.com/19277301/47033553-2646fa80-d143-11e8-8623-ca3bfa87b551.png)

**Version of Mod using:
Minecraft: 1.12.2
Forge: 14.23.5.2768
Extra Planets: 1.12.2 - 0.4.3
GalacticraftCore, Galacticraft Planets & MicdoodleCore: 1.12.2-4.0.1.181

**Where did this bug happen:** Single Player (I have not yet test this issue on server, but as soon as possible i will test and keep you informed)

### Additional Information
If you need more informations, justy tell me and I will be able to send you more informations!

Thank's for your amazing mod
Fireztonez

**Comment by MJRLegends** [comment]
Yeah its not meant to be enchanted. So i just need to block that it seems

**Comment by MJRLegends** [comment]
Im going to add some more modules to cover some of the enchantments but in a different way so make it something new

**Comment by Fireztonez** [comment]
Ok,
I will look that, thank's for your reply and the fix!
